pgrx-tests/src/framework.rs

```diff
@@ -312,7 +312,7 @@ fn install_extension() -> eyre::Result<()> {
         .arg("--pg-config")
         .arg(pg_config.path().ok_or(eyre!("No pg_config found"))?)
         .stdout(Stdio::inherit())
-        .stderr(Stdio::piped())
+        .stderr(Stdio::inherit())
```

Comment:
maybe out of place in this PR, but I burned well over an hour trying to figure out why the `cargo-pgrx` output was missing, leading me to believe schema generation wasn't even happening during `cargo test`.

Reply:
Any diagnostics changes necessary to uhhh write the PR are appropriate, I think, barring like a completely rewrite of framework.rs